src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -115,6 +115,10 @@ abstract class AbstractFunctionalTest extends Specification {
         generateRandomUUID()
     }
 
+    protected String createUniqueNetworkName() {
```

Comment:
Did this to be consistent with container names, which also have an abstraction that simply delegates to `generateRandomUUID` right above this method.